containers/wordpress/wp_base/https/Dockerfile

```diff
@@ -0,0 +1,83 @@
+ARG baseimage=wp_base
+FROM $baseimage
+
+LABEL authors="nitin.tekchandani@intel.com,lin.a.yang@intel.com,uttam.c.pawar@intel.com"
+
+USER root
+
+RUN apt-get update -y && \
+    apt-get remove nginx -y && \
+    apt-get install -y \
+    libpcre3 \
+    libpcre3-dev
+
+
+# Build/Install openssl v1_1_1f
```

Comment:
Should we be using OpenSSL_1_1_1j

Reply:
This was based on @uttampawar's internal reference but if no need to stay with f we can certainly move to j.